src/linux/mod.rs

```diff
@@ -86,6 +86,78 @@ pub const DT_REG: c_uchar = 8;
 pub const DT_LNK: c_uchar = 10;
 pub const DT_SOCK: c_uchar = 12;
 
+// include/linux/net.h
+#[cfg(not(any(target_arch = "mips", target_arch = "mips64")))]
+pub const SOCK_STREAM: c_int = 1;
```

Comment:
Please move all these constants with architecture specific values into their respective arch modules. (e.g. mod aarch64). There should be not `cfg` on constants in this file.